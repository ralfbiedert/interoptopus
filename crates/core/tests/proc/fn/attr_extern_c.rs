use interoptopus::ffi;

#[ffi]
extern "C" fn foo() {}

fn main() {}
