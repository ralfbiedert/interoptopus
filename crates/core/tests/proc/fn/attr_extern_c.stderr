error: Functions with explicit extern declarations are not supported. Remove the declaration and let #[ffi] handle it.
 --> tests/proc/fn/attr_extern_c.rs:4:1
  |
4 | extern "C" fn foo() {}
  | ^^^^^^^^^^
